core/arch/arm/plat-totalcompute/fdts/optee_sp_manifest.dts

```diff
@@ -0,0 +1,37 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2021, Arm Limited. All rights reserved.
+ *
+ * This file is a Partition Manifest (PM) for a minimal Secure Partition (SP)
```

Comment:
I suppose this comment should be updated to mention that it's a manifest for an OP-TEE as SP.

Reply:
Modified comments to mention OP-TEE as SP